README.md

```diff
@@ -1,10 +1,97 @@
 # Spa-Retreat-Project
 
-Spa Retreat website designed for project 3 of University Bootcamp
+Our Spa retreat project is based on our make believe spa '404GETAWAY'. A group application designed for University Bootcamp.
 
-## Make sure to do these 2 steps BEFORE writing any code!
+## User Story
+
+- As a user I want to be able to view the facilities at '404GETAWAY.
+- As a user I want to be able to create a secure account.
+- As a user I want to be able to log in to my account, edit my details, and see my wishlist.
+- As a user I want to be able to view and add treatments to my basket, and checkout securely.
+- As a user I want to be able to save treatments to view at a later time in my wishlist.
+
+## Table of Contents
+
+[User Story](#user-story)  
+[Technologies Used](#technologiesdependencies-used)  
+[Installation](#installation)  
+ [Usage](#usage)  
+[Screenshots](#screenshots)  
+[Contributing](#contributing)  
+[Deployed Link](#deployed-link)  
+[Licence](#licence)  
+[Future Development](#future-development)  
+[Contact Us](#contributing)
+
+## Technologies/Dependencies Used
+
+Back end
```

Comment:
no need to separate frontend and backend I think.. 
mention the MERN stack (i.e. MongoDB, Express.js, React.js, Node.js) first and del nodemon maybe